README.md

```diff
@@ -1,2 +1,3 @@
 # CprogrammingCodeWithHarry
 C Programming | Code With Harry
+this repository is best for beginner coders who wants to learn C language. 
```

Comment:
Why You have added this...Explain me!

Reply:
well, I checked your repository and it contain pretty much everything needed by beginner and I added this because I am learning how to use git hub. I saw a video of code with harry Youtube channel. I learnt this trick from there only. so can you please accept it. If you don't like it then you can change it after accepting it but for now please accept it. It would very motivating for me.